DataPlotly/core/plot_types/plot_type.py

```diff
@@ -85,12 +85,14 @@ def create_layout(settings):
             xaxis={
                 'title': x_title,
                 'autorange': settings.layout['x_inv'],
-                'range': range_x
+                'range': range_x,
+                'gridcolor': settings.layout['gridcolor']
```

Comment:
```suggestion
                'gridcolor': settings.layout.get('gridcolor', '#BDBFC0')
```
Older plots where this setting is not set, throw an error here. Solution is to set the default value